components-styled/StyledListWrapper.jsx

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+
+const StyledListWrapper = styled.div`
+  display: grid;
+  gap: 0.5rem;
```

Comment:
Do you want this gap to become triple the size if a person with lower sight capability cranks the default font size up to 300%? If not I would suggest you to use px here.

Reply:
I looked at it...my whole layout is with rem. Excitingly the font steps out of the layout but the gap doesn't change...in terms of my clientele this was a use case I rather didn't consider.... maybe also something to discuss after Tuesday